examples/src/examples/misc/gizmos.mjs

```diff
@@ -57,7 +57,7 @@ function controls({ observer, ReactPCUI, React, jsx, fragment }) {
                     jsx(BooleanInput, {
                         type: 'toggle',
                         binding: new BindingTwoWay(),
-                        link: { observer, path: 'gizmo.legacyRotation' }
+                        link: { observer, path: 'gizmo.useLegacyRotation' }
```

Comment:
I think it'd be better to have a more descriptive name then legacy - this does not help people decide which one they want to use.

Reply:
Yea Im just trying to think of a concise way to describe the difference.